get-dash/index.html

```diff
@@ -332,47 +332,49 @@ <h2 class="section__title">Use Dash everywhere</h2>
 						</div>
 						<h3 class="info-grid__item-title">{% t pages.get-dash.shop-heading %}</h3>
-						<p class="info-grid__item-text">{% t pages.get-dash.shop-text %}</p>
+						<span class="info-grid__item-text">{% t pages.get-dash.shop-text %}</span>
 						<a href="{% t pages.get-dash.shop-link %}" target="_blank" class="info-grid__item-link">{% t pages.get-dash.shop-btn %}</a>
 					</article>
 					<article class="info-grid__item" data-mh="get-dash-info-grid" id="getdash-visa">
 						<div class="info-grid__item-icon">
 							<svg width="56" height="59" viewBox="0 0 56 59" ><title>dash-label</title><g fill="none" fill-rule="evenodd"><path d="M53.854 11.17l-7.328-1.433-2.385-.47c-.548-2.49-2.382-6.72-3.805-8.778C39.52-.7 36.798.5 37.615 1.68c1.052 1.533 2.69 5.146 3.194 6.985.692 2.47.926 5.11.916 7.648l.003.366c-.015 1.09-.407 4.59-.69 5.688-.342 1.37 2.434 2.013 2.965.575.296-.777.836-4.322.86-5.203.05.036.1.08.153.125a5.79 5.79 0 0 1 1.82 4.598l-.004.027c-.013.14-.023.28-.043.427-.465 3.178-3.438 5.387-6.66 4.928-3.212-.46-5.443-3.41-4.98-6.59a5.763 5.763 0 0 1 2.5-3.96c.347-.243.582-.352.92-.51.046-2.608-.193-5.86-1.178-8.746-1.212-.232-2.72.136-3.7.87L.892 33.144A2.225 2.225 0 0 0 .44 36.25l16.216 21.43c.734.96 2.15 1.163 3.13.447l32.252-23.83c.984-.735 1.89-2.326 2.022-3.54L55.828 13.8c.128-1.213-.762-2.404-1.974-2.633" fill="#555"/><path d="M23.162 27.712l8.92 10.335c.69 1.043.477 2.48-.94 3.195-.87.44-2.04.846-3.175 1.306-1.147.463-2.33 1.04-3.17 1.242-1.185.284-3.027.08-4.198-1.07-1.52-1.49-2.947-3.413-4.46-5.168-1.53-1.772-3.062-3.486-4.398-5.162.947-.41 1.896-.814 2.835-1.138l7.903 9.155c1.88-.88 3.914-1.517 5.684-2.407a316.18 316.18 0 0 1-7.872-9.12c.954-.36 1.914-.783 2.87-1.168" fill="#FFF"/><path d="M18.115 28.606l4.7 5.445c-.91.343-1.82.714-2.73 1.05l-4.73-5.48c.917-.302 1.85-.777 2.76-1.014" fill="#FFF"/></g></svg>
 						</div>
 						<h3 class="info-grid__item-title">{% t pages.get-dash.visa-heading %}</h3>
-						<p class="info-grid__item-text">{% t pages.get-dash.visa-text %}</p>
+						<span class="info-grid__item-text">{% t pages.get-dash.visa-text %}</span>
```

Comment:
The file is massive, should be split into partials

Reply:
Maybe not as part of this PR

Reply:
@stpdenis maybe next time, not urgent.

The rest is fixed